rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -236,7 +236,7 @@ class rocalTensor {
     int create_from_handle(vx_context context);
     int create_virtual(vx_context context, vx_graph graph);
     bool is_handle_set() { return (_vx_handle != 0); }
-    void set_dims(std::vector<size_t>& dims) { _info.set_dims(dims); }
```

Comment:
why is this changed?

Reply:
Since we pass the rvalue of the vector, we will not be able to accept this vector by reference.
`set_dims( {_labels_id[i].size()} )`